web-registry/src/types/project/vcs-project.ts

```diff
@@ -0,0 +1,69 @@
+// TODO: Use [json-to-ts](https://marketplace.visualstudio.com/items?itemName=MariusAlchimavicius.json-to-ts) to generate type from
+// https://github.com/regen-network/regen-registry-standards/blob/a2d81baebe7756226a58b126378d1890ad4cadcc/jsonld/projects/C01-verified-carbon-standard-project.json
+
+export interface VcsProjectMetadataLD {
+  '@context': {
+    schema: 'http://schema.org/';
+    regen: 'http://regen.network/';
+    qudt: 'http://qudt.org/schema/qudt';
+    unit: 'http://qudt.org/2.1/vocab/unit';
+    xsd: 'http://www.w3.org/2001/XMLSchema#';
+  };
```

Comment:
this context is useless here since you're using full IRIs in the rest of the data, except for qudt and xsd, which should be fixed

Reply:
removed and fixed